packages/fork-choice/src/forkChoice/forkChoice.ts

```diff
@@ -214,10 +217,21 @@ export class ForkChoice implements IForkChoice {
           root: fromHexString(headRoot),
         });
       }
-
-      return (this.head = toBlockSummary(headNode));
+      prevHead = this.head;
+      return (this.head = newHead = toBlockSummary(headNode));
     } finally {
       if (timer) timer();
+      if (newHead && this.metrics) {
+        //if there is no prevHead, then isSameHead is false and isSameChain is true
+        const isSameHead = prevHead && toHexString(newHead.blockRoot) == toHexString(prevHead.blockRoot);
+        const isSameChain = !prevHead || isSameHead || this.isDescendant(prevHead.blockRoot, newHead.blockRoot);
+
+        if (isSameChain && !isSameHead) this.metrics.forkChoiceNewHeads.inc();
+        if (!isSameChain) this.metrics.forkChoiceNewChains.inc();
```

Comment:
This "newChains" metric is non-standard nor suggested by Lighthouse. The proper term is `reorg_total` and we are already tracking it somewhere else